MD/md-38/README.md

```diff
@@ -0,0 +1,76 @@
+# MD-38: Provide for Fixed Token Supply when Using Atomic Bridge and Fast Finality Settlement (FFS)
+- **Description**: Requests for Movement Improvement Proposals (MIPs) to provide for fixed token supply when using the Atomic Bridge and Fast Finality Settlement (FFS).
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Reviewer**: Andreas Penzkofer
+
+
+## Overview
+![Fixed Supply Rewarding Problem](./fixed-supply-rewarding-problem.png)
+
+
+Using Fast Finality Settlement with the Atomic Bridge presents a challenge when the token supply is fixed. 
+
+Naively burning token on the L2 without a way to return it to either pools on the L2 or the L1 will result in a **deflationary token**, i.e., the supply decreases over time and eventually reaches 0 rendering any operations relying on gas impossible. Meanwhile, providing minting capabilities which involve the operation of fallible services can result in **either a deflationary or inflationary token** and the opportunity for **byzantine attacks**.
+
+Namely, in this context, trying to use the same token for both L2 gas and L1 rewards requires that the token be bridged back over or else rely on a trusted signer to relay the gas. This document outlines the challenges and potential solutions for this problem.
+
+Users of the Movement SDK should be **informed of different approaches and trade-offs when using the Atomic Bridge and Fast Finality Settlement with a fixed token supply.** This document provides an initial conceptual framing towards that end. 
+
+This document also seeks a response to the concerns raised in [MD-20](https://github.com/movementlabsxyz/MIP/pull/20).
+
```

Comment:
```suggestion

## Definitions

- LP : Liquidity P???: 
- `$LP` : the LP token 

```

Reply:
abbreviations should be defined (unless they are in the glossary, which LP is not)